### DISCUSSION TITLE
Demote ManagerSerach to Manager

### DISCUSSION THREAD

**Discussion by brazorfajeje** [Security-Onion-Solutions/securityonion#5804] [Unsupported Versions] [answered]
Hi all!
I have a ManagerSearch and a SearchNode, I noticed that the managersearch node is processing data faster (5 times more EPS) than the pure search node, causing the /nsm partition of the managersearch to saturate faster than the searchnode one and causing incosistencies in the retention of the indexed logs.
I was wondering if it was possible to demote managersearch to a pure manager (I plan to add an additional searchnode anyway).

Thanks

**Comment by brazorfajeje** [comment, marked as answer]
For the record, 
I modified the managersearch grains replacing so-managersearch with so-manager (in /etc/salt/grains), 
renamed the minion in /etc/salt/minion for consistency, 
restarted the minion, added the key with salt-key 
copied/edited some configuration files in /opt/so/saltstack/local/,
applied highstate and restarted logstash

It seems to be working smoothly

**Reply by northben** [reply]
thanks for sharing! I just did the opposite, moving a manager to a managersearch node.

I also moved the minion file, I think this was a necessary step:
`mv /opt/so/saltstack/local/pillar/minions/somanager_manager.sls /opt/so/saltstack/local/pillar/minions/somanager_managersearch.sls`
